_pages/davidjgithubtraining.yaml

```diff
@@ -0,0 +1,4 @@
+---
+  pageUrl: https://davidjgithubtraining.github.io/supreme-octo-bassoon/
+  timestamp: 2017-03-30
+---
```

Comment:
@davidjgithubtraining make sure the format matches that specified in the course exactly: 

```
---
githubHandle: YOUR-USERNAME
pageUrl: YOUR-HANDLE.github.io
timestamp: TODAYS-DATE-YYYY-MM-DD
---
```